package.json

```diff
@@ -22,6 +22,7 @@
     "fast-glob": "^3.2.5",
     "glob": "^7.1.6",
     "got": "^11.8.0",
+    "hpagent": "^0.1.1",
```

Comment:
I selected [hpagent](https://www.npmjs.com/package/hpagent) since it fairly new as compared to some of the others recommend on got wiki. If there is a preferred alternate and extends http.Agent or https.Agent, it should be easy to swap out.